docs/overview_and_guide.rst

```diff
@@ -152,9 +155,12 @@ Then launch Raiden with the default testnet keystore path::
 Using parity
 ************
 
-Run the client and let it sync with the Ropsten testnet::
+Run the client and let it sync::
+
+    parity
```

Comment:
we don't need any additional arguments for parity? I.e. perhaps warn about warp sync here?

Reply:
There is a warning below, but I'll add the `--no-warp` to the command.